tidal_per_transformers/transformers/generic_recommendation_formatter_transformer.py

```diff
@@ -13,24 +15,24 @@ def __init__(self,
                  group_key: str,
                  recommendations_col: str,
                  item_type: str,
-                 module_name: str,
-                 module_version: str):
+                 module_name: str):
         super(GenericRecommendationFormatterTransformer, self).__init__()
         self.group_key = group_key
         self.recommendations_col = recommendations_col
-        self.item_type = item_type
+        self.item_type = item_type.upper()
         self.module_name = module_name
-        self.module_version = module_version
+        assert self.item_type in [x.upper() for x in [c.ALBUM, c.ARTIST, c.PLAYLIST, c.TRACK, c.MIX, c.RADIO,
+                                                      "ARTIST_TOP_TRACKS", "MASTER_BUNDLE_ID", "TRACK_GROUP"]]
```

Comment:
using upper case with under score for the supported types

Reply:
NIT: Should: `"ARTIST_TOP_TRACKS", "MASTER_BUNDLE_ID", "TRACK_GROUP"` be in constants here?

Reply:
Will `"MASTER_BUNDLE_ID"` be a valid type for the BE ?